docs/admin/collecting-metrics/README.md

```diff
@@ -43,7 +43,7 @@ In the following example, you can configure a single collector instance using a
 1. Create a Deployment, Service, and ConfigMap for the collector by entering the following command:
 
        ```bash
-       kubectl apply -f https://raw.githubusercontent.com/knative/docs/main/docs/admin/collecting-metrics/collector.yaml
+       kubectl apply -f https://raw.githubusercontent.com/knative/docs/master/docs/admin/collecting-metrics/collector.yaml
```

Comment:
this isn't right afaik, we use the main branch now?

Reply:
cc @skonto if there's any metrics stuff that's still valid and isn't on the `main` branch it should move

Reply:
The file also exists here: https://raw.githubusercontent.com/knative/docs/main/docs/admin/collecting-metrics/collector.yaml. We use main now. I am not aware of anything to be moved.